### ISSUE TITLE
Error when copy pasting txt into  a journal

### ISSUE THREAD

**Issue by SilverwolfNL** [foundryvtt/foundryvtt#9330]
### What happened?

whn trying to copy paste bunch of txt to a journal cant save journal . get an error "can not read properties of undefined (reading 0 )
![errorjournal](https://user-images.githubusercontent.com/26388997/235519943-826ec090-c340-4908-9266-dc2ec4a6aa18.png)

### What ways of accessing Foundry can you encounter this issue in?

- [X] Native App (Electron)
- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Other

### Reproduction Steps

open up a journl txt page. select txt from notepad and copy paste it to journal. gies an error message and cant save journal

![errorjournal](https://user-images.githubusercontent.com/26388997/235519465-86d2543d-b0b1-4459-98e0-fec4882810fb.png)

### What core version are you reporting this for?

v 11

### Bug Checklist

- [X] The issue occurs while all Modules are disabled

**Comment by Fyorl** [comment]
Are you able to confirm whether you updated to v11 via the in-app update or by downloading and installing a fresh version from the website?

**Comment by SilverwolfNL** [comment]
Via the in app update

**Comment by SilverwolfNL** [comment]
ok redownloaded v11 client as installer instead of updating via app ran that . then issue seems to be gone

**Comment by SilverwolfNL** [comment]
Addendum. can now indeed copy paste text.. but no longer to roll any dice in foudry. unistalled ans reinstalled v11 . same . dice button no longer works. also cant roll initative or od anything that requires rolling dice. will play arond with it tonight or this weekend. guess maybe need to do a reboot after uninstall before reinstalling

**Comment by SilverwolfNL** [comment]
Hmm altough now cant roll initative in combat tracker.. weird..pushign dice in combat tracker dos come with pop up. but clicking roll doe snothing

**Comment by Fyorl** [comment]
Can you verify that you have those issues with all modules disabled? And if you still have any scenes that exhibit the issue, or a backup with them, can you export one of them and attach it here?

**Comment by SilverwolfNL** [comment]
Hi, yes had all disabled.  tried other Scene i had in that world seems to work mostly fine. Mostly as roll idnivudal initiative works. roll all initative works.  Rolll NPC  does nothing. and roll all iitative did not work after i rested initatove. But did notmanage to replicate that. But rol iNPC initatove doe snot seem to work on any scenes. Not sure of thats a known buf

**Comment by SilverwolfNL** [comment]
in your gmail mailbbox now, titel of this tread as subject

**Comment by Fyorl** [comment]
I got it, thanks

**Comment by SilverwolfNL** [comment]
if i create a brand new world and then a scene it  seems ot work fine.. maybe because it was an existing world ocnverted to 11

**Comment by Fyorl** [comment]
Yeah, probably, but we still want to make sure that worlds migrated from v10 or earlier still function correctly in v11, so I will investigate your scene to see if there's something we missed.

**Comment by Fyorl** [comment]
@SilverwolfNL It looks like a scene won't be enough to diagnose your issue, because if I import it into my world, I don't have any of the NPC actors you are using, so I can't check why rolling initiative does not work. Can you send me your entire world instead? If you have a version before it was migrated to v11 that would be ideal, but I can still check your v11 migrated world if not.

**Comment by SilverwolfNL** [comment]
will send you a mail with oendrive link to wedensday.zip which is migrated   and wednesday-copy whihc is copy of fodler before it was migrated

**Comment by SilverwolfNL** [comment]
ok should come in your gmail now

**Comment by Fyorl** [comment]
That's great, I've got them, thanks

**Comment by Fyorl** [comment]
Hi again, I'm not able to reproduce your issue with the world you sent me:

https://github.com/foundryvtt/foundryvtt/assets/207433/52c2a323-ca30-4eb0-9b85-1047fb9550f1

You can see that I have no issue rolling initiative for any of the NPCs.

If you take your backed-up copy of the world and re-open it in the latest v11, do you still have issues rolling initiative for NPCs? If not, it might just be the case that your world ended up in a bad state with a buggy migration that we have since fixed.

If you still experience the issue after re-migrating the world, and with no modules enabled, then the only thing I could think of is perhaps issues with your dnd5e system itself. If you could confirm what dnd5e system version you're using as well in that case.

**Comment by aaclayton** [comment]
Closing as nonrepro
